### ISSUE TITLE
Thoughts on making library compatible with DynamoDb Document Client?

### ISSUE THREAD

**Issue by hoggkm** [rh389/dynamodb-geo.js#14]
I was going to do this myself becasue I would prefer to give / recieve data drom dynamo in a normalized JSON format rather than explicitly delcaring the types for everything, but I havent ot around to learning how to create my own modules yet..

https://aws.amazon.com/blogs/developer/announcing-the-amazon-dynamodb-document-client-in-the-aws-sdk-for-javascript/

I actually made some very basic changes to the local files after installing to make this work already and it is great... Didn't know if yuo were planning to add support for this at all.

**Comment by bognix** [comment]
I agree, I did some changes locally as well. Maybe we can join our efforts and make it work?

**Comment by rh389** [comment]
I don't have time to work on this right now but I'd love to see it - the current API feels very clunky and non-JS compared to the document client.

**Comment by igorescobar** [comment]
@hoggkm @bognix I started something here: #28. Let me know if you want to help.

**Comment by sichiu** [comment]
I'd like to see this so that I can use **convertEmptyValues**
